README.md

```diff
@@ -128,6 +128,19 @@ vars:
     quickbooks_union_databases: ['quickbooks_usa','quickbooks_canada'] # use this if the data is in different databases/projects but uses the same schema name
 ``` 
 
+### Configuring Account Type Names
+QuickBooks does allow for the `account_type` and `account_subtype` field values to be configured for customers internally. While our current filtered logic within our intermediate models accounts for the default values, it's possible there are different or multiple values that might change what the account types would be. 
+
+If you have different field value filters you'd like to set, you will need to configure the `account_type` and `account_subtype` variables that account for these variables in your `dbt_project.yml`.
```

Comment:
This is a bit misleading. We do reference account_type and account_subtype in multiple locations throughout the package, but this update specifically handles cases where we are referencing a specific account_type/subtype to be used in the double entry models. I would rephrase this to specifically call out this scenario. Something along the following (which you partly cover in the yml block below):

```suggestion
Within a few of the double entry models in this package a mapping takes place to assign certain transaction type's debits/credits to the appropriate offset account (ie. Accounts Payable, Accounts Receivable, Undeposited Funds, and SalesOfProductIncome) reference. While our current filtered logic within our intermediate models account for the default values, it's possible your use case relies on different account types to reference. 

If you have a different value to reference for each type, you will need to configure the `account_type` and `account_sub_type` variables that account for these variables in your `dbt_project.yml`.
```

Reply:
Updated!